### ISSUE TITLE
Scroll bar appears when left nav is visible but TOC is not

### ISSUE THREAD

**Issue by kwalrath** [dart-lang/site-www#244]
On both www & webdev, if you're on a page with a TOC, you'll see a bottom scrollbar if the TOC is not visible but the sidenav on the left _is_. Examples of pages where this occurs are https://www.dartlang.org/guides/language/analysis-options and https://webdev.dartlang.org/angular/guide.

For example, here's the problem at https://www.dartlang.org/guides/language/analysis-options:

![image](https://cloud.githubusercontent.com/assets/2164483/23087049/f58200fc-f525-11e6-9f6d-37f9cdddd7ff.png)

If the window's just a little wider or narrower, then it switches to a 3-column or 1-column layout with no bottom scrollbar.

**Comment by chalin** [comment]
I'm not seeing this scroll bar anymore on either site, are you @kwalrath?

**Comment by kwalrath** [comment]
Yes, I still see it, in both Chrome Canary Version 71.0.3556.0 and Chrome Version 69.0.3497.100. It doesn't seem the matter what the Zoom level is.
